resources/v2/01_students.sql

```diff
@@ -0,0 +1,20 @@
+CREATE TABLE students
+(
+    id SERIAL NOT NULL PRIMARY KEY,
+    name text NOT NULL,
+    city text NOT NULL,
+    age integer NOT NULL,
+    
+    created_at timestamp
+    WITH time zone DEFAULT NOW(),
+    subject_id integer NOT NULL
+    updated_at timestamp
+    WITH time zone,
+    deleted_at timestamp
+    WITH time zone
+);
+
+CREATE INDEX students_id ON students USING btree (id)
```

Comment:
You will never have to index the primary key.

Reply:
Got it. Updated accordingly.